src/components/VtkObliqueView.vue

```diff
@@ -0,0 +1,699 @@
+        default:
+          return ViewTypes.XY_PLANE;
+      }
+    });
+
+    const { baseImageRep } = useSceneBuilder<
+      vtkReslicReperesentationProxy
+    >(viewID, {
+      baseImage: curImageID
+    });
+
+    onBeforeMount(() => {
+      // do this before mount, as the ManipulatorTools run onMounted
+      // before this component does.
+      viewProxy.value.getInteractorStyle2D().removeAllManipulators();
+    });
+
+    const updateViewFromResliceCursor = () => {
+      const rep = baseImageRep?.value;
+      const resliceCursor = resliceCursorRef?.value;
+      const state = resliceCursor?.getWidgetState() as ResliceCursorWidgetState;
+      if (resliceCursor && rep) {
+        const planeOrigin = state.getCenter();
+        const planeNormal = resliceCursorRef.value.getPlaneNormalFromViewType(VTKViewType.value);
+        rep.getSlicePlane().setNormal(planeNormal);
+        rep.getSlicePlane().setOrigin(planeOrigin);
+        if (curImageData.value) {
+          resliceCursorRef.value.updateCameraPoints( viewProxy.value.getRenderer(), VTKViewType.value, false, false, true);
+        }
+      }
+    }
+
+    const onPlanesUpdated = useVTKCallback(
+      resliceCursorRef.value.getWidgetState().onModified
+    );
+
+    onPlanesUpdated(() => {
+      updateViewFromResliceCursor();
+    });
+
+    onMounted(() => {
+      setViewProxyContainer(vtkContainerRef.value);
+      viewProxy.value.setOrientationAxesVisibility(false);
+
+      // Initialize camera points during construction
+      if (curImageData.value) {
+        resliceCursorRef.value.updateCameraPoints(viewProxy.value.getRenderer(), VTKViewType.value, true, false, true);
+      }
+    });
+
+    onBeforeUnmount(() => {
+      setViewProxyContainer(null);
+    });
+
+    // --- apply windowing and slice configs --- //
+
+    function roundTo2Decimals(x: number): number {
+      return Math.round(x * 100) / 100;
+    }
+
+    // Function to compute float range of slicing for oblique slicing.
+    // Range is calculated as distance along the plane normal (as originating from {0,0,0} ).
+    function slicePlaneRange(cornerPoints: number[][], sliceNormal: number[]): [number, number] {
+      if (!cornerPoints || !sliceNormal)
+        return [0, 1];
+
+      // Get rotation matrix from normal to +X (since bounds is aligned to XYZ)
+      const transform = vtkMatrixBuilder
+        .buildFromDegree()
+        .identity()
+        .rotateFromDirections(sliceNormal, [1, 0, 0]);
+
+      const corners = cornerPoints.map(x => x.slice());
+      corners.forEach((pt) => transform.apply(pt));
+
+      // range is now maximum X distance
+      let minX = Infinity;
+      let maxX = -Infinity;
+      for (let i = 0; i < 8; i++) {
+        const x = corners[i][0];
+        if (x > maxX) {
+          maxX = x;
+        }
+        if (x < minX) {
+          minX = x;
+        }
+      }
+
+      return [roundTo2Decimals(minX), roundTo2Decimals(maxX)];
```

Comment:
What's the purpose of rounding to 2 decimal points here?

Reply:
Initially, I was trying to have some "meaningful" numbering for the oblique slices, but I don't think quantifying that is useful for non-axis-aligned slices. I am also thinking of removing the individual reset-camera buttons as they actually affect all three views. The global button on the toolbar should be sufficient.

Reply:
The rounding was since the numbers were displayed in the overlay.